### PR TITLE
Add app token e2e

### PR DESCRIPTION
**Pull request by elyezer**
##### SUMMARY

Add app token e2e

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - UI

##### AWX VERSION

```
awx: 6.1.0
```

### CODE REVIEW

File: awx/ui/test/e2e/tests/test-applications.js [hunk 1]
```diff
@@ -74,6 +90,61 @@ module.exports = {
...
+                .click('@tokensTab')
+                .waitForSpinny();
+            // FIXME: Search is not properly working, update this when it is working.
+            applications.section.tokens.expect.element('@list').text.to.contain(`${store.adminUser.username}\n${token.description}\nEXPIRATION ${tokenInfo.expires}`);
+        });
+
+        client.logout();
+        client.login();
+    },
     'delete an application': client => {
```
Comment (unlikelyzero) [inline awx/ui/test/e2e/tests/test-applications.js:148]: do we want to delete both tokens here?

Comment (elyezer) [reply awx/ui/test/e2e/tests/test-applications.js:148]: We can't delete tokens, see https://github.com/ansible/awx/issues/4446

**Comment by unlikelyzero** [comment]
This is awesome. If this can run in yolo 10x without failing, let's :shipit:

**Comment by elyezer** [comment]
@unlikelyzero I've run this couple times and the only test failing is the one releated to the websockets. We are fine to merge this I think.

**Comment by unlikelyzero** [comment]
@elyezer 🚢

**Review by unlikelyzero — APPROVED** [review]

**Pull request merged**